docs/guides/Checkout/customization/displaying-sla-by-item-in-checkout.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Displaying SLA by item in Checkout"
+slug: "displaying-sla-by-item-in-checkout"
+hidden: false
+createdAt: ""
+updatedAt: ""
+excerpt: "This guide describes how to display individual shipping estimates for items in a customer's cart on an ecommerce platform using the Checkout API."
+hidePaginationPrevious: false
+hidePaginationNext: false
+---
+
+To enhance the customer experience, some stores may want to display precise shipping estimates for each item in the customer's cart. This can be particularly useful for ecommerce platforms that have sellers with different shipping capabilities and delivery times. This guide will walk through the process of displaying the Service Level Agreement (SLA) by item using the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api).
+
+<CH.Scrollycoding>
+
+## Enable the `useIndividualShippingEstimates` flag
+
+First, it is necessary to enable the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api) behavior to fill the `shippingEstimateDate` with the SLA information. To do so, you must update the store's `orderForm` with the [Update orderForm configuration](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pvt/configuration/orderForm) endpoint. You must include the flag `useIndividualShippingEstimates` to `true` as in the example shown.
+
+```json Request mark=23
+{
+    "paymentConfiguration": {
+    ...
+    },
+    "taxConfiguration": null,
+    "minimumQuantityAccumulatedForItems": 1,
+    "decimalDigitsPrecision": 2,
+    "minimumValueAccumulated": null,
+    "apps": [
+    ...
+    ],
+    "allowMultipleDeliveries": true,
+    "allowManualPrice": null,
+    "savePersonalDataAsOptIn": false,
+    "maxNumberOfWhiteLabelSellers": null,
+    "recaptchaValidation": "vtexcriteria",
+    "recaptchaMinScore": null,
+    "recaptchaKeys": null,
+    "maskStateOnAddress": true,
+    "enableSecureCookies": true,
+    "useOwnershipCookie": true,
+    "ignoreProfileData": null,
+    "useIndividualShippingEstimates": true
+}
+```
+
+By enabling this flag, the `shippingEstimateDate` will be updated with the SLA information configured by the sellers.
+
+---
+
+## Add items to cart
+
+Once you enable the `useIndividualShippingEstimates`, test if the configuration is correct by adding items to a shopping cart using the [Add cart items](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/items) endpoint. See the request body example presented.
```

Comment:
"Once you enable the `useIndividualShippingEstimates` flag, test..."

"See the request body example below."